packages/headless/src/controllers/triggers/headless-execute-trigger.ts

```diff
@@ -22,13 +24,22 @@ export interface ExecuteTrigger extends Controller {
 export interface ExecuteTriggerState {
   /**
    * The name of the function to be executed.
+   *
+   * @deprecated
```

Comment:
```suggestion
   * @deprecated Use .... instead.
```

would be more helpful to the implementer